src/winetricks

```diff
@@ -2341,6 +2344,28 @@ _EOF_
 
 #---- Private Functions ----
 
+winetricks_dl_warning() {
+    case $LANG in
+    ru*) _W_countrymsg="Скрипт определил, что ваш IP адрес принадлежит России. Если во время загрузки файлов вы увидите ошибки несоответствия сертификата, скачайте файлы вручную или используйте VPN."
+      ;;
+    *)  _W_countrymsg="Your IP address has been determined to belong to Russia. If you encounter a certificate error while downloading, please use VPN or download files manually."
```

Comment:
The message should be changed from:
"Your IP address has been determined to belong to Russia. If you encounter a certificate error while downloading, please use VPN or download files manually."
to:
"Your IP address has been determined to belong to Russia. If you encounter a certificate error while downloading, please make sure tor is installed and relaunch with the --torify option, or use a VPN to download files manually."
becaue of https://github.com/Winetricks/winetricks/issues/630.  (Also, the Russian translation should be updated accordingly.)